server/server.go

```diff
@@ -390,6 +390,7 @@ func (s *Server) UnixListenAndServe() error {
 	return errors.New("can't listen on empty path")
 }
 
+// Close listeners
```

Comment:
Comments should be complete sentences (golint might complain about this). I think maybe this should be

```
Close shuts down the listeners.
```